libs/components/src/lib/selectable-box/selectable-box.spec.ts

```diff
@@ -159,11 +159,12 @@ describe('vwc-selectable-box', () => {
 				element = (await fixture(
 					`<${COMPONENT_TAG} control-type="radio" checked></${COMPONENT_TAG}>`
 				)) as SelectableBox;
-				const control = getControlElement(element);
-				const controlElement =
-					control.shadowRoot?.querySelector('[role="radio"]');
+				//const control = getControlElement(element);
+				const controlElement = element.querySelector(
+					'[role="radio"]'
+				) as HTMLElement;
```

Comment:
I think that will be the radio now:
```suggestion
				const controlElement = getControlElement(element);
```